firebase-project/src/router/index.js

```diff
@@ -0,0 +1,33 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import homeView from '../views/home.vue'
+
+const routes = [
+  {
+    path: '/',
+    name: 'home',
+    component: homeView
+  },
+  {
+    path: '/register',
+    name: 'registerPage',
```

Comment:
name: 'registerPage', 
route name should be dashed case `register-page` @mukeshsin